packages/common/src/Adaptor.js

```diff
@@ -267,12 +267,32 @@ export function asData(data, state) {
  * the state's references.
  * @public
  * @function
- * @example
- * each("$.[*]",
- *   create("SObject",
- *     field("FirstName", sourceValue("$.firstName"))
+ * @example <caption>Inserting patitent data using lazy state. (Only in v2)</caption>
+ * each($.data,
+ *   insert("patient",
+ *     {
+ *       patient_name: $.data.properties.case_name,
+ *       patient_id: $.data.case_id
+ *     }
  *   )
  * )
+ * @example <caption>Inserting patitent data with custom transformations. (Only in v1)</caption>
+ * each(
+ *   $.data,
+ *   insert("patient", (state) => ({
+ *     patient_id: state.data.case_id,
+ *     patient_name: state.data.properties.case_name.toUpperCase(),
```

Comment:
I think there are too many props here which bloat the example. Maybe I'd just set `patient_id` and then spread `state.data`

Reply:
I don't understand what this mean, can you elaborate more ?